dotCMS/src/main/java/com/dotmarketing/startup/runonce/Task04200CreateDefaultVanityURL.java

```diff
@@ -0,0 +1,235 @@
+		fields.add(setFieldElements("Uri", "com.dotcms.contenttype.model.field.TextField","", "text3", true, true, true, "uri", 2, "", "", "", "", true, false, true));
+		
+		fields.add(setFieldElements("Site","com.dotcms.contenttype.model.field.CustomField","","text2", true, true, true,"site",1,"$velutil.mergeTemplate('/static/content/site_selector_field_render.vtl')","","","",true,false,false));
+		
+		fields.add(setFieldElements("Title", "com.dotcms.contenttype.model.field.TextField","", "text1", true, true, true, "title", 1, "", "", "", "", true, false, true));
+
+		return fields;
+	}
+	
+	/**
+	 * This method sets the value of each element that is needed when creating a Field.
+	 * 
+	 * @param fieldName
+	 * @param fieldType
+	 * @param fieldRelationType
+	 * @param fieldContentlet
+	 * @param required
+	 * @param listed
+	 * @param indexed
+	 * @param velocityVarName
+	 * @param sortOrder
+	 * @param values
+	 * @param defaultValue
+	 * @param checkRegex
+	 * @param fixed
+	 * @param readOnly
+	 * @param searchable
+	 * @return
+	 */
+	private Map<String, Object> setFieldElements(String fieldName, String fieldType, String fieldRelationType,
+			String fieldContentlet, boolean required, boolean listed, boolean indexed, String velocityVarName,
+			int sortOrder, String values, String checkRegex, String hint, String defaultValue,boolean fixed, boolean readOnly,
+			boolean searchable) {
+
+		Map<String, Object> elements = new HashMap<>();
+		elements.put("structureInode", DEFAULT_VANITY_URL_STRUCTURE_INODE);
+		elements.put("fieldName", fieldName);
+		elements.put("fieldType", fieldType);
+		elements.put("fieldRelationType", fieldRelationType);
+		elements.put("fieldContentlet", fieldContentlet);
+		elements.put("isRequired", required);
+		elements.put("isIndexed", indexed);
+		elements.put("isListed", listed);
+		elements.put("velocityVarName", velocityVarName);
+		elements.put("sortOrder", sortOrder);
+		elements.put("values", values);
+		elements.put("regexCheck", checkRegex);
+		elements.put("hint", hint);
+		elements.put("defaultValue", defaultValue);
+		elements.put("isFixed", fixed);
+		elements.put("isReadOnly", readOnly);
+		elements.put("isSearchable", searchable);
+		elements.put("isUnique", false);
+		
+		return elements;
+	}
+
+	@Override
+	public boolean forceRun() {
+		return true;
+	}
+
+	@Override
+	public String getPostgresScript() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public String getMySQLScript() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public String getOracleScript() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public String getMSSQLScript() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	public String getH2Script() {
+		// TODO Auto-generated method stub
+		return null;
+	}
+
+	@Override
+	protected List<String> getTablesToDropConstraints() {
+		// TODO Auto-generated method stub
+		return null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Return an empty collection instead of null. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1168)